movie_search/models.py

```diff
@@ -18,3 +21,35 @@ class Provider(models.Model):
     def __str__(self):
         """String for representing the Model object."""
         return f"{self.provider_id}: {self.name}"
+
+
+class Video(models.Model):
+   
+    name = models.CharField(max_length=200)
+    key =  models.CharField(max_length=200)
+
+    def __str__(self):
+        """String for representing the Model object."""
+        return f"{self.name}: {self.key}"
+
+class Movie(models.Model):
+    movie_id = models.PositiveSmallIntegerField(default=0)
+    title = models.CharField(max_length=200, null=True, blank=True)
+    backdrop_path = models.CharField(max_length=200, null=True, blank=True)
+    genres = models.ManyToManyField(Genre, related_name="movies", blank=True)
+    tagline = models.CharField(max_length=200, null=True, blank=True)
+    vote_count = models.PositiveSmallIntegerField(default=0)
+    vote_average = models.PositiveSmallIntegerField(default=0)
+    popularity = models.IntegerField(default=0, null=True, blank=True)
```

Comment:
is this 1-10? then you could add something like `validators=[MaxValueValidator(10), MinValueValidator(1)]`

Reply:
vote count is an integer, and vote average and popularity are termed "numbers" in the API docs. Here is an example response. I think vote average and popularity should change to models.FloatField() right?
```
{  
  "poster_path": "/IfB9hy4JH1eH6HEfIgIGORXi5h.jpg",  
  "adult": false,  
  "overview": "Jack Reacher must uncover the truth behind a major government conspiracy in order to clear his name. On the run as a fugitive from the law, Reacher uncovers a potential secret from his past that could change his life forever.",  
  "release_date": "2016-10-19",  
  "genre_ids": [  
    53,  
    28,  
    80,  
    18,  
    9648  
  ],  
  "id": 343611,  
  "original_title": "Jack Reacher: Never Go Back",  
  "original_language": "en",  
  "title": "Jack Reacher: Never Go Back",  
  "backdrop_path": "/4ynQYtSEuU5hyipcGkfD6ncwtwz.jpg",  
  "popularity": 26.818468,  
  "vote_count": 201,  
  "video": false,  
  "vote_average": 4.19  
}
```

Reply:
oh then my suggestion does not match, yes seems more like `models.FloatField()`s then.